index.html

```diff
@@ -540,20 +545,34 @@ <h2 class="title">Get Start New Job</h2>
                 src="./src/backgound-svg/dots-design.svg"
                 alt="dots"
               />
+              <div class="sign-up-graphics__wrapper">
+                <div class="sign-up-graphics__content">
+                  <img
+                    class="sign-up-graphics__vector-vector-1"
```

Comment:
```suggestion
                    class="sign-up-graphics__vector-1"
```

Why do you need "vector-vector" in the name of the class?